src/combinatorics/simple_graph/clique.lean

```diff
@@ -48,12 +51,21 @@ by { simp_rw is_clique_iff, exact set.pairwise.mono' h }
 lemma is_clique.subset (h : t ⊆ s) : G.is_clique s → G.is_clique t :=
 by { simp_rw is_clique_iff, exact set.pairwise.mono h }
 
+@[simp] lemma is_clique_bot_iff : (⊥ : simple_graph α).is_clique s ↔ (s : set α).subsingleton :=
+set.pairwise_bot_iff
+
+alias is_clique_bot_iff ↔ simple_graph.is_clique.subsingleton _
+
 end clique
 
+/-! ### `n`-cliques -/
+
+section n_clique
 variables {n : ℕ} {s : finset α}
 
 /-- A `n`-clique in a graph is a set of `n` vertices which are pairwise connected. -/
```

Comment:
I had suggested this correction in the first PR, at least in the module docstring, but I notice it's still "which." Please change the "which" there to "that" as well.
```suggestion
/-- A `n`-clique in a graph is a set of `n` vertices that are pairwise connected. -/
```

Reply:
"pairwise adjacent" is better at both places IMO.